library/postgres

```diff
@@ -16,12 +16,12 @@ Directory: 16/bullseye
 
 Tags: 16.0-alpine3.18, 16-alpine3.18, alpine3.18, 16.0-alpine, 16-alpine, alpine
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 7442464585e3cd75554976cbe94819a42da10bbd
+GitCommit: 2bff0ce33b76f03f4e6986497e0ab084dad10e9d
```

Comment:
Similar to https://github.com/docker-library/official-images/pull/15474#pullrequestreview-1660880825, these should be https://github.com/docker-library/postgres/commit/6f4ae836406b010948f01fbcb400a31dca4fdf52